frontend/src/features/navigation/footer/footer.tsx

```diff
@@ -0,0 +1,70 @@
+import { Box, Divider, Grid, Link, styled, useTheme } from '@mui/material';
+
+import FacebookIcon from '@mui/icons-material/Facebook';
+import InstagramIcon from '@mui/icons-material/Instagram';
+
+const Footer = () => {
+  const theme = useTheme();
+  return (
+    <FooterContainer
+      sx={{
```

Comment:
why sx prop in here together with styled? It would be better to move all styles to styled component below